src/components/layouts/App.js

```diff
@@ -153,19 +155,55 @@ function App() {
               </Routes>
             </Suspense>
           </ErrorBoundary>
-        ) : (
+        </main>
+      ) : (
+        <main className="auth-container">
           <ErrorBoundary FallbackComponent={ErrorFallbackAuth}>
             <Suspense fallback={<Loader header=" " spinner />}>
-              <Routes>
-                <Route element={<SignIn />} path="/" />
-                <Route element={config.maintenance ? <Maintenance /> : <SignUp />} path="/sign-up" />
-                <Route element={isMaintenance ? <Maintenance /> : <ResetPassword />} path="/reset-password" />
-                <Route path="*" element={<Navigate to={`/?redirect=${pathname}${search}`} replace />} />
-              </Routes>
+              <Container fluid="xs" className="h-100">
+                <Row xs="1" sm="2" className="h-100">
+                  <Col className="p-5 auth-studio-info d-none d-sm-flex justify-content-center align-items-center">
+                    <div className='auth-studio-info-container'>
+                      <h1 className="auth-title">HarperDB Studio</h1>
+                      <span className="mb-4 auth-subtitle d-inline-block">Manage all your HarperDB instances.</span>
+                      <ul className="auth-info-list-items">
+                        <li>
+                          <h3 className="item-title">Manage All Instances</h3>
+                          <span className="item-subtitle">Set access, cluster, monitor, and more.</span>
+                        </li>
+                        <li>
+                          <h3 className="item-title">Embedded API Server</h3>
+                          <span className="item-subtitle">Custom Functions give you unlimited application flexibility.</span>
```

Comment:
Probably should say "HarperDB components" instead of "Custom Functions"

Reply:
Thanks!